File: Assets/AltUnityTester/AltUnityDriver/Commands/EnableDebugging.cs
public class EnableDebugging : AltBaseCommand
{
    public EnableDebugging(SocketSettings socketSettings) : base(socketSettings)
    {
    }
    public void Execute(){
        Socket.Client.Send(toBytes(CreateCommand("enableDebug", SocketSettings.DebugFlag.ToString())));
        var data = Recvall();
        if (data.Equals("Ok"))
            return;
        HandleErrors(data);
    }
}

File: Assets/AltUnityTester-BindingsAndExamples(can_be_deleted)/Bindings~/java/src/main/java/ro/altom/altunitytester/Commands/EnableDebugging.java
package ro.altom.altunitytester.Commands;

import ro.altom.altunitytester.AltBaseSettings;

public class EnableDebugging extends AltBaseCommand {
    public EnableDebugging(AltBaseSettings altBaseSettings) {
        super(altBaseSettings);
    }
    public String Execute(){
        send(CreateCommand("enableDebug", altBaseSettings.debugEnabled.toString()));
        String data = recvall();
        if (data.equals("OK")) {
            return data;
        }
        handleErrors(data);
        return "";
    }
}

